### PR TITLE
Fetch the default picture if a product has no image defined

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Fetch the default picture if a product has no image defined
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes PrestaShop/PrestaShop#28131
| How to test?      | Cf. PrestaShop/PrestaShop#28131

### CODE REVIEW

File: src/Calculator/StatisticsCalculator.php [hunk 1]
```diff
@@ -200,9 +200,13 @@ public function getProductImage($productDetails)
 
         foreach ($presentedProduct as $key => $value) {
             if ($key == 'embedded_attributes') {
+                dump($value);
```
Comment (NeOMakinG) [inline src/Calculator/StatisticsCalculator.php:203]:
```suggestion
```

Comment (Progi1984) [reply src/Calculator/StatisticsCalculator.php]: Fixed

File: src/Calculator/StatisticsCalculator.php [hunk 2]
```diff
@@ -149,6 +149,7 @@ public function computeConversionRate(&$stats, $dateStart = null)
             }
 
             $imgDetails = $this->getProductImage($productDetails);
+            dump($imgDetails);
```
Comment (matthieu-rolland) [inline src/Calculator/StatisticsCalculator.php:152]:
```suggestion
```

**Review by matthieu-rolland — COMMENTED** [review]
dumb should be removed, lgtm otherwise...

File: src/Calculator/StatisticsCalculator.php (hunk 2 above)
Comment (Progi1984) [reply src/Calculator/StatisticsCalculator.php]: Fixed

**Comment by Progi1984** [comment]
@matthieu-rolland Feedback addressed

**Review by NeOMakinG — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @AureRita

**Pull request merged**